### PR TITLE
Add changelog for v5.8.0

### PR DESCRIPTION
**Pull request by moskyb**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v5.8.0](https://github.com/buildkite/elastic-ci-stack-for-aws/tree/v5.8.0) (2022-02-28)
+[Full Changelog](https://github.com/buildkite/elastic-ci-stack-for-aws/compare/v5.7.2...v5.8.0)
+
+### Added
```
Comment (pda) [inline CHANGELOG.md:10]: Previous entries put a blank line between the heading and the lists.
I don't know if there's markdown renderers that care about the difference, though.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
...
+- Allow Docker address pool customisation [#968](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/968) (@ouranos)
```
Comment (pda) [inline CHANGELOG.md:11]: This one ended up being more like:

> Smaller docker networks allow 16x as many docker-compose networks

We removed the customisation aspect mentioned in the PR title.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
...
+- Add support for additional ARM/Graviton instance types, reprise [#981](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/981) (@toothbrush)
+- Recognise c7g instances as ARM [#979](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/979) (@pda)
```
Comment (pda) [inline CHANGELOG.md:13]: These two could be rolled into one, with credit to @toothbrush and @yob

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
...
+- Bump buildx from 0.5.1 to 0.7.1 [#975](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/975) (@orien)
+- Quieten Fixing permissions header log group [#965](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/965) (@keithduncan)
+- Update issue templates [#947](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/947) (@keithduncan)
+- Update agent version to v3.43.0 [#990](https://github.com/buildkite/pull/990) (@moskyb)
+
+### Security
+- Create SECURITY.md [#948](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/948) (@keithduncan)
+
+### Fixed
+- fix: force symlink override if exists [#970](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/970) (@chefsale)
```
Comment (pda) [inline CHANGELOG.md:28]: This one could be:

> Overwrite /usr/bin/buildkite-agent symlink if it already exists

**Review by eleanorakh — CHANGES_REQUESTED** [review]

**Review by eleanorakh — APPROVED** [review]
I've also reviewed pauls suggestion above, I think that #968 is fine.

Other suggestion non-blocking 💃

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,32 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v5.8.0](https://github.com/buildkite/elastic-ci-stack-for-aws/tree/v5.8.0) (2022-02-28)
+[Full Changelog](https://github.com/buildkite/elastic-ci-stack-for-aws/compare/v5.7.2...v5.8.0)
+
+### Added
+
+- Smaller docker networks allow 16x as many docker-compose networks [#968](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/968) (@ouranos)
```
[the remark this answers is not in the document]
Comment (moskyb) [reply CHANGELOG.md]: i think that part of the PR never went through, all we did is customise the docker network sizes, and we didn't end up adding the flag. i've updated this summary to be a little clearer though, regardless

**Review by pda — APPROVED** [review]

**Pull request merged**